gadgets/testing/testing.go

```diff
@@ -36,6 +36,13 @@ const (
 	K8sDistroGKECOS         = "gke-COS_containerd"
 )
 
+const (
+	BusyBoxImage          = "ghcr.io/inspektor-gadget/ci/busybox:latest"
+	NginxImage            = "ghcr.io/inspektor-gadget/ci/nginx:latest"
+	GccImage              = "ghcr.io/inspektor-gadget/ci/gcc:latest"
+	NetworkMultitoolImage = "ghcr.io/inspektor-gadget/ci/network-multitool:latest"
+)
```

Comment:
```suggestion
	BusyBoxImage          = "ghcr.io/inspektor-gadget/inspektor-gadget/ci/busybox:latest"
	NginxImage            = "ghcr.io/inspektor-gadget/inspektor-gadget/ci/nginx:latest"
	GccImage              = "ghcr.io/inspektor-gadget/inspektor-gadget/ci/gcc:latest"
	NetworkMultitoolImage = "ghcr.io/inspektor-gadget/inspektor-gadget/ci/network-multitool:latest"
)
```

ref: https://github.com/inspektor-gadget/inspektor-gadget/pull/4122#issuecomment-2679651259

Reply:
Resolved: https://github.com/inspektor-gadget/inspektor-gadget/pull/4133